client/login/wp-login/index.jsx

```diff
@@ -58,6 +60,16 @@ export class Login extends React.Component {
 
 	static defaultProps = { isJetpack: false, isGutenboarding: false, isLoginView: true };
 
+	state = {
```

Comment:
We store the username in this parent component and pass it as a prop to `LoginLinks`.